purchase_order_line_from_stock/models/stock.py

```diff
@@ -31,6 +43,7 @@ def add_purchase_order_line_from_stock_move(self):
                     "qty_received": stock_move.product_qty,
                     "qty_invoiced": stock_move.product_qty,
                     "price_unit": stock_move.product_id.list_price,
+                    "taxes_id": [(6, 0, [stock_move.product_id.taxes_id.id])],
```

Comment:
do the purchase order will recompute the sub_total and the total amount of the purchase order ?

Reply:
Jes, it does! But I had to fix this line to manage multiple taxes. Thanks for drawing my attention to it!
![Screen Shot 2020-11-23 at 13 22 35](https://user-images.githubusercontent.com/2920134/99961694-22105100-2d8f-11eb-8b54-f91ad52c0f36.png)